Homework3/task2.py

```diff
@@ -9,4 +10,4 @@
 for i in range(int(len(list) / 2 + 0.5)):
     lst.append(list[i] * list[(len(list) - i - 1)])
```

Comment:
```suggestion
    lst.append(list[i] * list[len(list) - i - 1])
```
Можно вообще убрать `len(list)` – результат будет тот же.

Reply:
А еще, кажется, я неоднократно повторял, что не стоит называть переменные именем типа данных, который содержат эта переменная.